examples/tools/system/package_manager/conanfile.py

```diff
@@ -0,0 +1,50 @@
+from conan import ConanFile
+from conan.tools.system import package_manager
+from conan.tools.gnu import PkgConfig
+
+required_conan_version = ">=2.0"
+
+
+class SysNcursesConan(ConanFile):
+    name = "ncurses"
+    version = "system"
+    description = "A textual user interfaces that work across a wide variety of terminals"
+    topics = ("curses", "terminal", "toolkit")
+    homepage = "https://invisible-mirror.net/archives/ncurses/"
+    license = "MIT"
+    package_type = "shared-library"
+    settings = "os", "arch", "compiler", "build_type"
+
+    def layout(self):
+        pass
+
+    def package_id(self):
+        self.info.clear()
+
+    def system_requirements(self):
+        if self.settings.os != "Linux":
+            return
+
+        dnf = package_manager.Dnf(self)
+        dnf.install(["ncurses-devel"], update=True, check=True)
+
+        yum = package_manager.Yum(self)
+        yum.install(["ncurses-devel"], update=True, check=True)
+
+        apt = package_manager.Apt(self)
+        apt.install(["libncurses-dev"], update=True, check=True)
+
+        pacman = package_manager.PacMan(self)
+        pacman.install(["ncurses"], update=True, check=True)
+
+        zypper = package_manager.Zypper(self)
+        zypper.install(["ncurses"], update=True, check=True)
+
+    def package_info(self):
+        self.cpp_info.bindirs = []
+        self.cpp_info.includedirs = []
+        self.cpp_info.libdirs = []
+
+        if self.settings.os != "Linux":
```

Comment:
But the ``system_requirements()`` didn't install anything, so what would it detect?

Reply:
Addressed by [1588288](https://github.com/conan-io/examples2/pull/177/commits/1588288ccab4c1c09e875191a15ad2669fddf2e9)